docs/application/dotnet/guides/nui/customview.md

```diff
@@ -82,9 +82,147 @@ To create a custom view:
     }
     ```
 
-    The following table lists other important custom view methods that you can use to manage the view.
+5. Define the constructor to create an instance of the custom view. In this case, the `ContactView` is created with the `contactName` parameter, and the resource directory path points to the icons. Additionally, the `base` constructor has to be called with the type name and view behavior as described in the subsequent section. In a nonstatic constructor, `Visuals` used to create a custom view should be created. For instance, in the following `ContactView` example, one label, background, and a few icons are shown:
 
-    **Table: Custom view methods**
+    ![Contact View](media/contact.png)
+
+    ```
+    public ContactView(string contactName, string resDir) : base(typeof(ContactView).Name, CustomViewBehaviour.ViewBehaviourDefault)
+        {
+            CreateBackground(new Vector4(1.0f, 1.0f, 1.0f, 1.0f));
+            CreateIcon(resDir + "/images/cbg.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_BG_ICON_INDEX);
+            CreateIcon(resDir + "/images/contact.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_ICON_INDEX);
+            CreateIcon(resDir + "/images/edit.png", 130.0f, 40.0f, 50.0f, 50.0f, CONTACT_EDIT_INDEX);
+            CreateIcon(resDir + "/images/favorite.png", 180.0f, 40.0f, 50.0f, 50.0f, CONTACT_FAVORITE_INDEX);
+            CreateIcon(resDir + "/images/delete.png", 640.0f, 40.0f, 50.0f, 50.0f, CONTACT_DELETE_INDEX);
+            CreateLabel(name);
+        }
+    ```
+
+6. Implementation of `Visuals` used in the `ContactView` example is shown in the following example:
+
+> [!IMPORTANT] 
+> It is recommended to get yourself familiarized with [Visuals](visuals.md) before you start implementing your own `CustomView`.
+
+   To start with, you need to define valid indexes for registering `Visuals` in the `CustomView`. These Indexes help you in setting a layer depth of `Visual` and bind it with the current view. In the `ContactView` example following indexes are used:
+
+    ```
+        private const int BASE_INDEX = 10000;
+        private const int BACKGROUND_VISUAL_INDEX = BASE_INDEX + 1;
+        private const int LABEL_VISUAL_INDEX = BASE_INDEX + 2;
+        private const int CONTACT_BG_ICON_INDEX = BASE_INDEX + 3;
+        private const int CONTACT_ICON_INDEX = BASE_INDEX + 4;
+        private const int CONTACT_EDIT_INDEX = BASE_INDEX + 5;
+        private const int CONTACT_FAVORITE_INDEX = BASE_INDEX + 6;
+        private const int CONTACT_DELETE_INDEX = BASE_INDEX + 7;
+    ```
+
+    In the next section `Visuals` creation is described:     
```

Comment:
![113](https://user-images.githubusercontent.com/43839737/99369271-390ff880-28e2-11eb-8e4c-a9ec6d8e9991.PNG)


- Just happened to check the page on staging, line 128 is rendering as code. 
- Request you to align the text 1-2 spaces to left such that the text becomes normal (As you already know by default 4 spaces is treated as code block on Github)


Is code styling at 122, 133. 135, 159, 161, etc needed? If not, can you please do the needful by removing them and just keep one at the start and one at the end? and If these are kept purposefully, then it is fine.

Reply:
Changed.